test/TEST-04-JOURNAL/test-journal.sh

```diff
@@ -74,6 +74,20 @@ cmp /expected /output
 { journalctl -ball -b  -m 2>&1 || :; } | head -1 > /output
 cmp /expected /output
 
+# https://github.com/systemd/systemd/issues/13708
+ID=$(journalctl --new-id128 | sed -n 2p)
+systemd-cat -t "$ID" bash -c 'echo parent; (echo child) & wait' &
+PID=$!
+wait %%
+journalctl --sync
+journalctl -b -o export -t "$ID" --output-fields=_PID | grep '^_PID=' >/output
```

Comment:
What do you mean when you say "this"? FWIW __CURSOR etc. have been output by the export format for a long time according to git blame.

Reply:
>  FWIW __CURSOR etc. have been output by the export format for a long time according to git blame.

Yeah, and I think this is a bug. They are written in a separate part of the code than the other fields (because they are based on the entry header, not data), and I think this was overlooked when `--output-fields=` was being implemented. But I don't think we can just change this at this point, because of backwards compatibility concerns.

Maybe we should add `--only-fields=` that would only write the specified fields.

Reply:
If you want to create an issue I can add a comment which references it.

Reply:
It seems we already have an issue: #5697.

Reply:
Never mind, #5697 is about a different issue.
I opened #13937 now.